### PR TITLE
Nav enhance 

### PR DESCRIPTION
**Pull request by collinnolen**
…me manager, board manager system. Also made the tiles hexagonal and the user can click to go to a new location.

### CODE REVIEW

**Comment by artsiriwatt** [comment]
@collinnolen Make sure to rebase this to the latest version of master, so that the diff is more manageable to read and to resolve conflicts

**Comment by artsiriwatt** [comment]
@collinnolen the branch still seems a bit funky, for example `loader.cs` is added in the first commit but excluded in the second commit, when in the git history it shouldn't have been included in the first commit.

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 # Visual Studio 2015 cache directory
 /.vs/
+/.vscode/
```
Comment (artsiriwatt) [inline .gitignore]: changes to .gitignore or config files that are unrelated should be done in a separate branch/pull request

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{
+    "files.exclude":
```
Comment (artsiriwatt) [inline .vscode/settings.json]: i'm not sure this file should be included either

File: Assets/Scripts/GameManager.cs [hunk 3]
```diff
@@ -8,6 +8,7 @@ public class GameManager : MonoBehaviour {
 
 	void Awake(){
        
+
```
Comment (artsiriwatt) [inline Assets/Scripts/GameManager.cs]: remove - we don't want extralines included in commits/merges unless specifically detailed for styling purposes

File: commit [hunk 4]
```diff
@@ -0,0 +1,35 @@
+Merge branch 'master' of https://github.com/jgayewu/pirategame into Nav_enchancements
```
Comment (artsiriwatt) [inline commit]: not sure what this file is

**Comment by collinnolen** [comment]
How do I into all commits, I dunno how to fix it at this point

On Dec 18, 2017 5:56 PM, "artsiriwatt" <notifications@github.com> wrote:

@collinnolen <https://github.com/collinnolen> the branch still seems a bit
funky, for example loader.cs is added in the first commit but excluded in
the second commit, when in the git history it shouldn't have been included
in the first commit.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/jgayewu/pirategame/pull/5#issuecomment-352615953>, or mute
the thread

File: .gitignore (hunk 1 above)
Comment (collinnolen) [reply .gitignore]: word

File: .vscode/settings.json (hunk 2 above)
Comment (collinnolen) [reply .vscode/settings.json]: it was included by mistake

File: commit (hunk 4 above)
Comment (collinnolen) [reply commit]: me trying and failing to do git stuff.

**Comment by collinnolen** [comment]
@artsiriwatt To clean everything up, how do I undo all of these commits but keep all the changes?

**Comment by artsiriwatt** [comment]
@collinnolen since some of the changes made in 65c2ad254a048bba1406fb2c7b203123b3fddd8c were already to merged to master, this is going to be a bit annoying to merge. The best option might be:
1. create a new branch based on the current version of master. 
2. cherry-pick commit 65c2ad254a048bba1406fb2c7b203123b3fddd8c. this is going to show some conflicts; resolve them and ensure the changes included are safe/don't break anything. 
3. cherrypick the three latest commits in the same order, checking that they include the correct changes and nothing breaks
